test/parallel/test-tls-sni-server-client.js

```diff
@@ -87,6 +87,13 @@ const clientsOptions = [{
 const serverResults = [];
 const clientResults = [];
 
+// check for throwing case where servername is not supplied
+common.expectsError(tls.createServer(serverOptions, () => {}).addContext, {
+  type: TypeError,
+  code: 'ERR_TLS_REQUIRED_SERVER_NAME',
+  message: '"servername" is a required parameter'
```

Comment:
The argument is called `hostname` in the docs, not sure what to use.

Reply:
If docs and source code disagree, one or the other should be changed so they are in agreement.